python-threatexchange/threatexchange/hashing/ocr_utils.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Util file for Optical character recognition (OCR) related functions.
+Use of pytesseract will required additional libaries, see https://github.com/madmaze/pytesseract#installation
```

Comment:
```suggestion
Use of pytesseract requires additional libaries already be installed, see https://github.com/madmaze/pytesseract#installation
```

Reply:
Thanks, for now I am going to leave it at this and update the readme in another PR that will provide context on this and the other libraries we added.